### PR TITLE
Added 60 Pre-con Commander Decks 

### PR DESCRIPTION
**Pull request by riku4470**
Added 60 Pre-con Commander Decks to the quest shop from the first set all the way through Kamaigawa Neon. Also updated PRE-CON IMAGES.TXT  so new images i submitted will be downloaded automatically. 

I submitted my images to Northmoc to upload

### CODE REVIEW

File: forge-gui/res/quest/precons/Arcane Maelstrom.dck [hunk 1]
```diff
@@ -0,0 +1,99 @@
+[metadata]
+Name=Arcane Maelstrom
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Cast instants to turn tide of any battle and summon creatures right at home in the middle of a magical storm.
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arcane Maelstrom.dck:9]: `turn tide` -> `turn the tide`

File: forge-gui/res/quest/precons/Arcane Maelstrom.dck [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+1 Temur Charm|C20
+1 Thornwood Falls|C20
+1 Tribute to the Wild|C20
+1 Twinning Staff|C20
+1 Whiplash Trap|C20
+1 Wilderness Reclamation|C20
+1 Wort, the Raidmother|C20
+1 Xyris, the Writhing Storm|C20
+1 Yavimaya Coast|C20
+[sideboard]
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arcane Maelstrom.dck:99]: probably unneeded?

File: forge-gui/res/quest/precons/Arcane Wizardry.dck [hunk 3]
```diff
@@ -0,0 +1,102 @@
+[metadata]
+Name=Arcane Wizardry
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Inalla, Archmage Ritualist delves into arcane mysteries in pursuit of forbidden dark magic. Gathering an inner circle of loyal acolytes to her cause. Inalla harnesses their collective power to extract the life force from her enemies.
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arcane Wizardry.dck:9]: `her cause. Inalla` -> `her cause, Inalla`

File: forge-gui/res/quest/precons/Arcane Wizardry.dck [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Name=Arcane Wizardry
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Inalla, Archmage Ritualist delves into arcane mysteries in pursuit of forbidden dark magic. Gathering an inner circle of loyal acolytes to her cause. Inalla harnesses their collective power to extract the life force from her enemies. 
+Set=C17
+Image=arcane_wizardy.png
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arcane Wizardry.dck:11]: `arcane_wizardy` -> `arcane_wizardry`

File: forge-gui/res/lists/precon-images.txt [hunk 5]
```diff
@@ -8,14 +8,18 @@ https://downloads.cardforge.org/images/products/precons/angelic_might.jpg
 https://downloads.cardforge.org/images/products/precons/angrath_minotaur_pirate.jpg
 https://downloads.cardforge.org/images/products/precons/anthousas_army.jpg
 https://downloads.cardforge.org/images/products/precons/arcane_tempo.jpg
+https://downloads.cardforge.org/images/products/precons/arcane_wizardy.png
```
Comment (Northmoc) [inline forge-gui/res/lists/precon-images.txt:11]: `arcane_wizardy` -> `arcane_wizardry`

File: forge-gui/res/lists/precon-images.txt [hunk 6]
```diff
@@ -74,31 +84,42 @@ https://downloads.cardforge.org/images/products/precons/depths_of_power.jpg
...
 https://downloads.cardforge.org/images/products/precons/elvish_rage.jpg
+https://downloads.cardforge.org/images/products/precons/elven_empire.png
 https://downloads.cardforge.org/images/products/precons/endless_march.jpg
+https://downloads.cardforge.org/images/products/precons/enhanced_evolution.png
 https://downloads.cardforge.org/images/products/precons/enlightened_mastery.jpg
 https://downloads.cardforge.org/images/products/precons/entangling_webs.jpg
+https://downloads.cardforge.org/images/products/precons/entropic_uprising.png
 https://downloads.cardforge.org/images/products/precons/esper_air_assault.jpg
 https://downloads.cardforge.org/images/products/precons/esper_artifice.jpg
+https://downloads.cardforge.org/images/products/precons/eternal_bargin.png
```
Comment (Northmoc) [inline forge-gui/res/lists/precon-images.txt:110]: `eternal_bargin` -> `eternal_bargain`

File: forge-gui/res/quest/precons/Arm for Battle.dck [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+1 Unquestioned Authority|CMR
+1 Valorous Stance|CMR
+1 Volcanic Fallout|CMR
+1 Wear // Tear|CMR
+1 White Sun's Zenith|CMR
+1 Wild Ricochet|CMR
+1 Wind-Scarred Crag|CMR
+1 Winds of Rath|CMR
+1 Word of Seizing|CMR
+[sideboard]
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arm for Battle.dck:93]: remove?

File: forge-gui/res/quest/precons/Aura of Courage.dck [hunk 8]
```diff
@@ -0,0 +1,103 @@
+[metadata]
+Name=Aura of Courage
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Stand tall next to Galea and arm your forces with powerful Auras and Equipment. With a paladin’s protection on your side, none shall do you harm.
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Aura of Courage.dck:9]: `paladin’s` -> `paladin's` (remove curly apostrophe)

File: forge-gui/res/quest/precons/Aura of Courage.dck [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+1 Utopia Sprawl|AFC
+1 Valiant Endeavor|AFC
+1 Valorous Stance|AFC
+1 Verdant Embrace|AFC
+1 Viridian Longbow|AFC
+1 Vitu-Ghazi, the City-Tree|AFC
+1 Wild Growth|AFC
+1 Winds of Rath|AFC
+1 Winged Boots|AFC
+[sideboard]
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Aura of Courage.dck:103]: remove

File: forge-gui/res/quest/precons/Aura of Courage.dck [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Stand tall next to Galea and arm your forces with powerful Auras and Equipment. With a paladin’s protection on your side, none shall do you harm.
+Set=AFC
+Image=aura_of_courage.png
+[commander]
+1 Galea, Kindler of Hope|AFC
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Aura of Courage.dck:13]: should this be foiled?

File: forge-gui/res/quest/precons/Arm for Battle.dck [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Wyleth may start small, but the Auras and Equipment in this deck can quickly make him an imposing battlefield presence. Every attack brings you a bounty of new cards to clear opposing forces, enhance your other creatures, or make Wyleth that much more dominating.
+Set=CMR
+Image=arm_for_battle.png
+[commander]
+1 Wyleth, Soul of Steel|CMR
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arm for Battle.dck:13]: foil?

File: forge-gui/res/quest/precons/Arcane Wizardry.dck [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+[Commander]
+1 Inalla, Archmage Ritualist|C17
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Arcane Wizardry.dck:13]: foil?

**Comment by Northmoc** [comment]
I think most (if not all) of the commanders should be foiled with the `+` right?

I think commander decks don't need `[sideboard]`, right?

File: forge-gui/res/quest/precons/Breed Lethality.dck [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+1 Deepglow Skate|C16
+1 Ikra Shidiqi, the Usurper|C16
+1 Duelist's Heritage|C16
+1 Grip of Phyresis|C16
+1 Ancient Excavation|C16
+1 Sylvan Reclamation|C16
+1 Manifold Insights|C16
+1 Migratory Route|C16
+1 Sublime Exhalation|C16
+1 Ash Barrens|C16
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Breed Lethality.dck:96]: add newline at end of file

File: forge-gui/res/quest/precons/Buckle Up.dck [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+1 Swift Reconfiguration|NEC
+1 Command Tower|NEC
+1 Exotic Orchard|NEC
+1 Port Town|NEC
+1 Prairie Stream|NEC
+1 Skycloud Expanse|NEC
+1 Spire of Industry|NEC
+1 Temple of Enlightenment|NEC
+15 Plains|NEC
+15 Island|NEC
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Buckle Up.dck:85]: add newline

**Comment by Northmoc** [comment]
Make sure all files have one blank newline at the end

File: forge-gui/res/quest/precons/Counterpunch.dck [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Use Ghave, Guru of Spores to create a strangling tangle of vines and vegetation that will crush all who oppose you! The all-consuming power of growth turns your creatures into monstrous behemoths while Saprolings fuel your increasing might.
+Set=CMD
+Image=counterpunch.png
+[commander]
+1 Ghave, Guru of Spores
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Counterpunch.dck:13]: looks like this probably needs `|CMD` after each card?

File: forge-gui/res/quest/precons/Coven Counters.dck [hunk 16]
```diff
@@ -0,0 +1,92 @@
+[metadata]
+Name=Coven Counters
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Elected as “Harvesttide Sovereign” by the witches of the Dawnhart Coven, Leinore has thrown a festival for the ages. Her hard work has filled both the witches and townsfolk of Innistrad with hope—perhaps this long night will soon see daybreak. Leinore hands out +1/+1 counters to everyone on her guest list. At the end of the night, your horde of humans will be the last ones standing.
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Coven Counters.dck:9]: `“Harvesttide Sovereign”` -> `"Harvesttide Sovereign"`

File: forge-gui/res/quest/precons/Devour for Power.dck [hunk 17]
```diff
@@ -0,0 +1,90 @@
+[metadata]
+Name=Devour for Power
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=When The Mimeoplasm becomes your commander, death becomes your friend. Opponents will watch in helpless terror as you devour creatures from one graveyard to help you pound creatures into another. It’s a fiendish carnival of cadavers, and you’ll be the ringmaster!
+Set=CMD
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Devour for Power.dck:10]: should this be COM, or should the rest of the cards be CMD?

File: forge-gui/res/quest/precons/Devour for Power.dck [hunk 18]
```diff
@@ -0,0 +1,90 @@
+[metadata]
+Name=Devour for Power
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=When The Mimeoplasm becomes your commander, death becomes your friend. Opponents will watch in helpless terror as you devour creatures from one graveyard to help you pound creatures into another. It’s a fiendish carnival of cadavers, and you’ll be the ringmaster!
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Devour for Power.dck:9]: `It’s a fiendish carnival of cadavers, and you’ll` -> `It's a fiendish carnival of cadavers, and you'll`

File: forge-gui/res/quest/precons/Draconic Rage.dck [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Harness the power of rage with Vrondiss, calling upon the spirits of his dragonborn ancestors to incinerate your enemies.
+Set=AFC
+Image=draconic_rage.png
+[commander]
+1 Vrondiss, Rage of Ancients|AFC
+[main]
+1 Anger
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Draconic Rage.dck:15]: add `|AFC`

File: forge-gui/res/quest/precons/Dungeons of Death.dck [hunk 20]
```diff
@@ -0,0 +1,98 @@
+[metadata]
+Name=Dungeons of Death
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Venture into the Forgotten Realms’ deepest dungeons with Sefris. Emerge with a powerful army of undead and drag your opponents to their doom.
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Dungeons of Death.dck:9]: `Realms’` -> `Realms'`

File: forge-gui/res/quest/precons/Enhanced Evolution.dck [hunk 21]
```diff
@@ -0,0 +1,99 @@
+[metadata]
+Name=Enhanced Evolution
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Overpower opponents with brand-new mechanic from Ikoria: Lair of Behemoths—mutate!
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Enhanced Evolution.dck:9]: `with brand-new mechanic` -> `with a brand-new mechanic` or `with the brand-new mechanic` ??

File: forge-gui/res/quest/precons/Eternal Bargain.dck [hunk 22]
```diff
@@ -0,0 +1,102 @@
...
+Name=Eternal Bargain
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Oloro, Ageless Ascetic manipulates the forces of life and death. This commander uses powerful lifegain abilities to fuel his pursuit of knowlege and dominance.
+Set=C13
+Image=eternal_bargin.png
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Eternal Bargain.dck:11]: `bargin` -> `bargain`

File: forge-gui/res/quest/precons/Feline Ferocity.dck [hunk 23]
```diff
@@ -0,0 +1,108 @@
...
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Arahbo, Roar of the World is a majestic leader and father to cats across all planes of the multiverse. His deafening voice empowers his ferocious feline allies to claw through enemy forces and crush those who dare challenge his ancient might. 
+Set=C17
+Image=feline_ferocity.png
+[Commander]
+1 Arahbo, Roar of the World|C17
+[Main]
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Feline Ferocity.dck:14]: lowercase in []

**Comment by Northmoc** [comment]
THANKS for doing all this legwork! I've given about a third of them a look; I'll look the rest over later.

**Comment by riku4470** [comment]
> 
Thanks for the review process i wasnt sure how this whole thing get done. Its pretty cool. So i can tell you right now i will need to remove the sideboard at the bottom of all the files and make all the commanders Foil. I will also have to check for that empty line at the bottom. 

Other then that i will correct anything else you got. Hopefully i am using GitHub correctly with the corrections.

File: forge-gui/res/lists/precon-images.txt (hunk 5 above)
Comment (riku4470) [reply forge-gui/res/lists/precon-images.txt]: done

File: forge-gui/res/lists/precon-images.txt (hunk 6 above)
Comment (riku4470) [reply forge-gui/res/lists/precon-images.txt]: done

File: forge-gui/res/quest/precons/Draconic Domination.dck [hunk 24]
```diff
@@ -0,0 +1,110 @@
+[metadata]
+Name=Draconic Domination
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
```
Comment (Northmoc) [inline forge-gui/res/quest/precons/Draconic Domination.dck:8]: So I'm noticing a number of these files have two different `[metadata]` lines... probably should delete the second one

Comment (paulsnoops) [reply forge-gui/res/quest/precons/Draconic Domination.dck:8]: I will merge this now and then make a PR to correct these

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**